tests/Unit/Service/OutOfOffice/OutOfOfficeParserTest.php

```diff
@@ -114,16 +114,16 @@ public function testBuildEnabledResponderWithoutEndDate(): void {
 	}
 
 	public function testBuildEnabledResponderWithSpecialCharsInMessage(): void {
-		$script = file_get_contents(__DIR__ . "/../../../data/sieve-vacation-cleaned.txt");
-		$expected = file_get_contents(__DIR__ . "/../../../data/sieve-vacation-on-special-chars-message.txt");
+		$script = file_get_contents(__DIR__ . "/../../../data/sieve-vacation-cleaned.sieve");
+		$expected = file_get_contents(__DIR__ . "/../../../data/sieve-vacation-on-special-chars-message.sieve");
 
 		$actual = $this->outOfOfficeParser->buildSieveScript(
 			new OutOfOfficeState(
 				true,
 				new DateTimeImmutable("2022-09-02T00:00:00+0100"),
 				null,
 				"On vacation",
-				"I'm on vacation.\n\"Hello, World!\"\n\\ escaped backslash",
+				"I'm on vacation. \"Hello, World!\" \\ escaped backslash",
```

Comment:
I fear we have to drop the newlines from the test for now, because mixing clrf lf in `sieve-vacation-on-special-chars-message.sieve` is not possible.

Reply:
Does that also mean that line breaks from the out-of-office feature will be lost when the autoresponder integration is used?

Reply:
Line breaks in the autoresponder mail body still work. 

The *.sieve files are stored with line ending CLRF in git. The test data uses LF. I didn't find a way (apart from using a hex editor) to store CLRF and LF in the same file. 

But I adjusted the test differently, that should also work.

Reply:
Ahh, I see, thanks. Fine by me then.